Modules/CalcPerform.lua

```diff
@@ -903,7 +906,7 @@ function calcs.perform(env)
 	end
 
 	-- Combine buffs/debuffs 
-	output.EnemyCurseLimit = modDB:Sum("BASE", nil, "EnemyCurseLimit")
+	output.EnemyCurseLimit = modDB:Sum("BASE", nil, "EnemyCurseLimit") + (output.AwakenedCurseOnHit or 0)
```

Comment:
I think instead of adding up all instances of `EnemyCurseLimit` with `output.AwakenedCurseOnHit`, it's more extensible to add all `EnemyCurseLimit` and `AdditionalCurse` instead. That way, you could implement new mechanics that conditionally grant additional curses without having to change that line.